lib/sanbase/billing/plan.ex

```diff
@@ -35,6 +60,10 @@ defmodule Sanbase.Billing.Plan do
     field(:is_deprecated, :boolean, default: false)
     # plans that customers can't subscribe on their own
     field(:is_private, :boolean)
+
+    # is plan for parity purchasing power
+    field(:is_ppp, :boolean, default: false)
```

Comment:
```suggestion
    @desc ~s"""
    Mark if this is a parity purchasing power power.
    In some countries our pricing is reduced in order to better match the purchasing power of the population.
    """
    field(:is_ppp, :boolean, default: false)
```

You can use `@desc` attribute so this comment will be seen while exploring the API via GraphiQL, making the doc visible outside of our codebase.